R/utilities-writing-report.R

```diff
@@ -198,24 +198,23 @@ renderReport <- function(fileName, createWordReport = FALSE, numberSections = TR
   return(invisible())
 }
 
-#' @title renderWordReport
-#' @description Render docx report with number sections and table of content
-#' @param fileName name of .md file to render
-#' @param intro name of .md file that include introduction (before toc)
-#' @param createWordReport option for creating Markdown-Report only but not a Word-Report
-#' @param wordConversionTemplate optional docx template for rendering a tuned Word-Report document
-#' @export
-#' @family reporting
-renderWordReport <- function(fileName, intro = NULL, createWordReport = FALSE, wordConversionTemplate = NULL) {
-  reportConfig <- file.path(reEnv$log$folder, "word-report-configuration.txt")
-  wordFileName <- sub(pattern = ".md", replacement = "-word.md", fileName)
-  docxWordFileName <- sub(pattern = ".md", replacement = "-word.docx", fileName)
-  docxFileName <- sub(pattern = ".md", replacement = ".docx", fileName)
-  fileContent <- readLines(fileName, encoding = "UTF-8")
-
-  wordFileContent <- NULL
-  figureContent <- NULL
-  for (lineContent in fileContent) {
+#' @title translateForWord
+#' @description
+#' Translate markdown text into a version that Pandoc can convert to Word with appropriate formatting
+#' @param mdText Character strings of markdown text
+#' @return Updated md text
+#' @keywords internal
+translateForWord <- function(mdText) {
+  wordText <- NULL
+  # Page break using source code
+  # https://pandoc.org/MANUAL.html#extension-raw_attribute
+  artifactPageBreak <- c(
+    "```{=openxml}",
+    '<w:p><w:r><w:br w:type="page"/></w:r></w:p>',
```

Comment:
Same as for the bookmarks: do we really need `<w:p><w:r>` here? 
If no: let's remove and keep only `<w:br w:type="page"/>`

Reply:
I checked, there is indeed no need for these tags